### ISSUE TITLE
GM 2 BAR MAP sensor values wrong

### ISSUE THREAD

**Issue by duhafnusa4** [rusefi/rusefi#3828]
the kpa and voltages for the default GM 2 BAR MAP sensor are actually the values for a GM 3 BAR sensor. 

Below are the Values for the 1,2 and 3 BAR sensors (we should make all 3 defaults) 

![image](https://user-images.githubusercontent.com/5051341/150662543-af9bd004-5bc4-4506-b0ae-b90a45d36a81.png)

**Comment by duhafnusa4** [comment]
Theres already one on the default list called "3 bar". It should just be re-named "GM 3 Bar", and then correct the values for the 1 and 2 BAR GM

**Comment by rusefillc** [comment]
@duhafnusa4 thank you! Just renamed "3 bar" to "GM 3 Bar"

**Comment by rusefillc** [comment]
kind of reminds me of #3054

**Comment by rusefillc** [comment]
@duhafnusa4 why do you think that 2 bar settings are wrong?

how do you know that? what are you doing where and how to see rusEFI not working properly?

reason I am asking for clarification is due to fact that code looks OK. I wonder if calibration is right but we have some sort of usability confusion

```
	case MT_GM_3_BAR:
		return &gm3bar;
	case MT_GM_2_BAR:
		return &gm2bar;
	case MT_GM_1_BAR:
		return &gm1bar;

static FastInterpolation gm3bar(0.631, 40, 4.914, 304);

static FastInterpolation gm2bar(0, 8.8, 5, 208);

static FastInterpolation gm1bar(0, 10, 5, 105);

```

**Comment by duhafnusa4** [comment]
Yeah kind of inconsistent. We dont need 4 billion different presets. People can always do custom if they want. I would say if you have a hellen board, then we should have defaults for that chassis' sensors

**Comment by rusefillc** [comment]
@duhafnusa4 you are starting to mix more and more sexy stuff into one overloaded issue

let's go back to original problem statement. what makes you report this issue? please elaborate

**Comment by duhafnusa4** [comment]
> @duhafnusa4 why do you think that 2 bar settings are wrong?
> 
> how do you know that? what are you doing where and how to see rusEFI not working properly?
> 
> reason I am asking for clarification is due to fact that code looks OK. I wonder if calibration is right but we have some sort of usability confusion
> 
> ```
> 	case MT_GM_3_BAR:
> 		return &gm3bar;
> 	case MT_GM_2_BAR:
> 		return &gm2bar;
> 	case MT_GM_1_BAR:
> 		return &gm1bar;
> 
> static FastInterpolation gm3bar(0.631, 40, 4.914, 304);
> 
> static FastInterpolation gm2bar(0, 8.8, 5, 208);
> 
> static FastInterpolation gm1bar(0, 10, 5, 105);
> ```

see the values for 1 bar:

![image](https://user-images.githubusercontent.com/5051341/150662813-410d7f06-9f15-4083-894a-395437090e27.png)

see the values for 2 bar:

![image](https://user-images.githubusercontent.com/5051341/150662825-87be1cc2-2293-4479-af57-ce2e580a31ca.png)

**Comment by rusefillc** [comment]
greyed out values do not mean shit. that's a usability issue - you are looking at greyed out values which are only use for 'custom' and you assume that greyed out values are applied, those are not

this would be addressed if we make progress with #3054 - that's a usability issue not functional issue.

**Comment by duhafnusa4** [comment]
even if greyed out we should be displaying the values for that sensor.

**Comment by rusefillc** [comment]
@duhafnusa4 I agree that current state of affairs is shit

**Comment by rusefillc** [comment]
@BeerMoneyMotorsports @OrchardPerformance @NMSTEC significant part of this change is rinse and repeat https://github.com/rusefi/rusefi/pull/3326/files
